soccer/gameplay/skills/goalside_mark.py

```diff
@@ -114,6 +103,11 @@ def mark_point(self) -> robocup.Point:
     @mark_point.setter
     def mark_point(self, value: robocup.Point):
         self._mark_point = value
+    
+    # Sets the position to mark as the given mark position 
+    # or robot position if no mark position is given
+    def _reset_mark_pos(self):
+        self.mark_pos = self.mark_point if self.mark_point is not None else self._mark_robot.pos
```

Comment:
Is it ever possible that `mark_point` will be None and `_mark_robot` will be None?

Reply:
I mean... mark_robot could be a string.  But specifically stated that one of these is required in the docstring